Assets/Scripts/PlayerTPLocationDefinition.cs

```diff
@@ -0,0 +1,9 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerTPLocationDefinition : MonoBehaviour

```

Comment:
Can you use the full word Teleport here? 
So `PlayerTeleportLocationDefinition`?